sktime/classification/deep_learning/tapnet.py

```diff
@@ -230,3 +230,35 @@ def _fit(self, X, y):
         )
 
         return self
+
+
+@classmethod
+def get_test_params(cls, parameter_set="default"):
+    """Return testing parameter settings for the estimator.
+
+    Parameters
+    ----------
+    parameter_set : str, default="default"
+        Name of the set of test parameters to return, for use in tests. If no
+        special parameters are defined for a value, will return `"default"` set.
+        For classifiers, a "default" set of parameters should be provided for
+        general testing, and a "results_comparison" set for comparing against
+        previously recorded results if the general set does not produce suitable
+        probabilities to compare against.
+
+    Returns
+    -------
+    params : dict or list of dict, default={}
+        Parameters to create testing instances of the class.
+        Each dict are parameters to construct an "interesting" test instance, i.e.,
+        `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+        `create_test_instance` uses the first (or only) dictionary in `params`.
+    """
+    return {"n_epochs": 100, "batch_size": 2, "dilation": 2}
```

Comment:
Wouldn't it make more sense to increase the `batch_size`? We could also try to decrease `n_epochs` to 50 and also decrease the default `filter_sizes` to maybe `(128, 128, 64)` & dense sizes (via `layers` to `(200, 100)`).

This network is bigger than others that are currently present here and these values come from paper's recommendation. As far as I understand, in these tests, we're not testing for convergence but for proper working. These values might not be optimal for convergence but should be good for testing.